sympy/physics/continuum_mechanics/beam.py

```diff
@@ -72,19 +73,26 @@ def __init__(self, length, elastic_modulus, second_moment, variable=Symbol('x'),
         ==========
         length : Sympifyable
             A Symbol or value representing the Beam's length.
+
         elastic_modulus : Sympifyable
             A SymPy expression representing the Beam's Modulus of Elasticity.
             It is a measure of the stiffness of the Beam material. It can
             also be a continuous function of position along the beam.
-        second_moment : Sympifyable
-            A SymPy expression representing the Beam's Second moment of area.
-            It is a geometrical property of an area which reflects how its
-            points are distributed with respect to its neutral axis. It can
-            also be a continuous function of position along the beam.
+
+        second_moment : Sympifiable or Geometry object
+            Describes the cross-section of the beam  via a sympy expression
+            representing the Beam's Second moment of area. It is a geometrical
+            property of an area which reflects how its points are distributed
+            with respect to its neutral axis. It can also be a continuous
+            function of position along the beam.
+            Or via a Geometry object representing the shape of the cross-section
+            of the beam.
```

Comment:
I would keep the above as it was and instead of the last sentence write:
```
Alternatively ``cross_section`` can be a shape object such as a ``Polygon``
from the geometry module. The second moment of area will be computed
from the shape object.
```
This should either be all one paragraph (no line break) or should have a blank line to separate the paragraphs.